mcs/class/Makefile

```diff
@@ -410,8 +410,14 @@ DISTFILES = \
 	doc/TemplateTest.cs	\
 	notes/BitVecto32.txt	\
 	MicrosoftAjaxLibrary/License.htm	\
-	test-helpers/NetworkHelpers.cs	\
-	test-helpers/SocketResponder.cs	\
+	test-helpers/AdminHelper.cs \
+	test-helpers/Configuration.Http.cs \
+	test-helpers/NetworkHelpers.cs \
+	test-helpers/NunitHelpers.cs \
+	test-helpers/PlatformDetection.cs \
+	test-helpers/RemoteExecutorTestBase.Mobile.cs \
+	test-helpers/RemoteExecutorTestBase.Mono.cs \
+	test-helpers/SocketResponder.cs \
```

Comment:
I thought we dist all the .cs files from .sources? Maybe that doesn't work for tests?

Reply:
I don't think DISTFILES/EXTRA_DIST can traverse _up_ a directory tree? mcs/build/tests.make assigns a full list o' files to TEST_FILES by parsing the relevant .sources files, but I think those files in test-helpers were only being included by being referenced elsewhere (e.g. the two above), not via the .sources parsing

```
directhex@flame:~/Projects/mono/mcs/class$ sed -e '/^$$/d' -e 's,^../,,' -e '/^\#.*$$/d' -et -e 's,^,Test/,' corlib/corlib_test.dll.sources | grep ^\\.\\.
../../../mono/mini/TestHelpers.cs
../test-helpers/NunitHelpers.cs
```